src/Services/NodeGeneralInfoService.ts

```diff
@@ -0,0 +1,34 @@
+import {NodeGeneralInfo} from "../Models/NodeGeneralInfo";
+
+export class NodeGeneralInfoService {
+
+    public async updateOrCreateNodeGenralInfo(
+        version: string, sectorSize: number, minerPower: number, totalPower: number, nodeId: number) {
+        const node = await NodeGeneralInfo.findOne({
```

Comment:
Why don't you use the below method `fetchNodeGenralInfo` rather than duplicate code?

Reply:
Old habits, still in the process of reverting to DRY.